index.d.ts

```diff
@@ -1,4 +1,4 @@
-declare function yn (value: string | number | boolean, options?: yn.Options): boolean | null;
+declare function yn (value: string | number | boolean | undefined, options?: yn.Options): boolean | null;
```

Comment:
please just make the parameter optional with a `?` like `options`

Reply:
Ok, thought it will make it more explicit, that it's not optional, but can receive `undefined`. Though technically they will behave exactly the same.